src/components/CoinsList.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import CoinItem from './CoinItem';
+import { fetchCoins } from '../redux/coins/coins';
+
+const CoinsList = () => {
+  const coins = useSelector((state) => state.coins);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if ((coins.length) === 0)dispatch(fetchCoins());
+  }, []);
+
+  const aCoins = coins.map((coin, i) => {
+    if (((i + 1) % 2) === 0) {
+      return { ...coin, checker: 'evenCoin', order: i + 1 };
+    }
+    return { ...coin, checker: 'oddCoin', order: i + 1 };
+  });
+
+  const oddCoins = [...aCoins].filter((coin) => coin.checker === 'oddCoin');
+
+  const oddMapped = oddCoins.map((coin, i) => {
+    if (((i + 1) % 2) === 0) {
+      return { ...coin, checker: 'oddCoin' };
+    }
+    return { ...coin, checker: 'evenCoin' };
+  });
+
+  const evenCoins = [...aCoins].filter((coin) => coin.checker === 'evenCoin');
+
+  const evenMapped = evenCoins.map((coin, i) => {
+    if (((i + 1) % 2) === 0) {
+      return { ...coin, checker: 'evenCoin' };
+    }
+    return { ...coin, checker: 'oddCoin' };
+  });
+
+  const classedCoins = evenMapped.concat(oddMapped);
+
+  const allCoins = [...classedCoins].sort((a, b) => a.order - b.order);
+
+  return (
+    <>
+      <div className="coinsHeader">
+        <h2>Crypto Coins</h2>
+      </div>
```

Comment:
- [ ] This is meant to be the navigation of your app. Please follow the design guidelines and include the back, settings, and mic icons 🙏 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/182952862-9006b752-5a84-4901-8420-a39a9c159901.png)
### Design Guideline
![image](https://user-images.githubusercontent.com/46839548/182952942-9bceda6e-a249-4928-a774-de4e1f1d2ddd.png)

- [ ]  [OPTIONAL] I think it would be a great idea to extract this into a reusable navigation component since you will be needing it in your `<Coin />` page which is a detail page 🤔